modules/custom/az_media/config/install/image.style.az_full_width_bg_large.yml

```
@@ -6,14 +6,14 @@ dependencies:
   module:
     - crop
 name: az_full_width_bg_large
-label: 'Large full-width background (Scales width down to 1920px)'
+label: 'Large full-width background (Scales width down to 1140px)'
 effects:
   97cf4f70-46ce-4581-85e0-d3b996645664:
     uuid: 97cf4f70-46ce-4581-85e0-d3b996645664
     id: image_scale
     weight: -9
     data:
-      width: 1920
+      width: 1140
```

```suggestion
      width: 1199
```
For background image, Josh and I think that they should stretch to 1 pixel below the upper limit of the [grid breakpoint](https://digital.arizona.edu/arizona-bootstrap/v5/docs/5.0/layout/grid/)
```
$grid-breakpoints: (
  xs: 0,
  sm: 576px, // so az_full_width_bg_extra_small should be 575 (it currently is)
  md: 768px, // so az_full_width_bg_small should be 767 (it currently is)
  lg: 992px,  // so az_full_width_bg_medium should be 991 (it currently is)
  xl: 1200px, // so az_full_width_bg_large should be 1199
  xxl: 1400px 
);
```